dist/utils/date.js

```diff
@@ -1,7 +1,12 @@
-"use strict";
+'use strict';
 
 function parseISOString(s) {
-  var b = s.split(/\D+/);
+  var b = s && s.split(/\D+/);
+
+  if (!b) {
+    return '';
+  }
+
```

Comment:
Why do we need this?

Reply:
when `s` is undefined, the function returns an error. Instead, since the caller is expecting a string value, I am returning  a string value in cases when `s` and `b` don't exist.